apphub/README.md

```diff
@@ -45,42 +45,42 @@ $ fastestimator train lenet_mnist.py --batch_size 64 --epochs 4
 
 ## Table of Contents:
 ### Natural Language Processing (NLP)
-* **Text Classification:** Sentiment classification based on IMDB movie reviews. [[paper](https://www.bioinf.jku.at/publications/older/2604.pdf)][[tensorflow code](https://github.com/fastestimator/fastestimator/blob/master/apphub/NLP/imdb/imdb_tf.py)][[pytorch code](https://github.com/fastestimator/fastestimator/blob/master/apphub/NLP/imdb/imdb_torch.py)][[notebook](https://github.com/fastestimator/fastestimator/blob/master/apphub/NLP/imdb/imdb.ipynb)]
+* **Text Classification:** Sentiment classification based on IMDB movie reviews. [[paper](https://www.bioinf.jku.at/publications/older/2604.pdf)][[tensorflow code](https://github.com/fastestimator/fastestimator/blob/r1.0/apphub/NLP/imdb/imdb_tf.py)][[pytorch code](https://github.com/fastestimator/fastestimator/blob/r1.0/apphub/NLP/imdb/imdb_torch.py)][[notebook](https://github.com/fastestimator/fastestimator/blob/r1.0/apphub/NLP/imdb/imdb.ipynb)]
 
-* **Named Entity Recognition**: BERT fine tuning on a German news corpora. [[tensorflow code](https://github.com/fastestimator/fastestimator/blob/master/apphub/NLP/named_entity_recognition/bert_tf.py)][[pytorch code](https://github.com/fastestimator/fastestimator/blob/master/apphub/NLP/named_entity_recognition/bert_torch.py)][[notebook](https://github.com/fastestimator/fastestimator/blob/master/apphub/NLP/named_entity_recognition/bert.ipynb)]
+* **Named Entity Recognition**: BERT fine tuning on a German news corpora. [[tensorflow code](https://github.com/fastestimator/fastestimator/blob/r1.0/apphub/NLP/named_entity_recognition/bert_tf.py)][[pytorch code](https://github.com/fastestimator/fastestimator/blob/r1.0/apphub/NLP/named_entity_recognition/bert_torch.py)][[notebook](https://github.com/fastestimator/fastestimator/blob/r1.0/apphub/NLP/named_entity_recognition/bert.ipynb)]
 
 ### Adversarial Training
-* **Fast Gradient Sign Method:** Adversarial training on the CIFAR-10 dataset using the Fast Gradient Sign Method. [[paper](https://arxiv.org/abs/1412.6572)][[tensorflow code](https://github.com/fastestimator/fastestimator/blob/master/apphub/adversarial_training/fgsm/fgsm_tf.py)][[pytorch code](https://github.com/fastestimator/fastestimator/blob/master/apphub/adversarial_training/fgsm/fgsm_torch.py)][[notebook](https://github.com/fastestimator/fastestimator/blob/master/apphub/adversarial_training/fgsm/fgsm.ipynb)]
+* **Fast Gradient Sign Method:** Adversarial training on the CIFAR-10 dataset using the Fast Gradient Sign Method. [[paper](https://arxiv.org/abs/1412.6572)][[tensorflow code](https://github.com/fastestimator/fastestimator/blob/r1.0/apphub/adversarial_training/fgsm/fgsm_tf.py)][[pytorch code](https://github.com/fastestimator/fastestimator/blob/r1.0/apphub/adversarial_training/fgsm/fgsm_torch.py)][[notebook](https://github.com/fastestimator/fastestimator/blob/r1.0/apphub/adversarial_training/fgsm/fgsm.ipynb)]
```

Comment:
aren't we going to have problems if the master branch links point to r1.0?

Reply:
we can have separate readme for r1.0 then. we don't want to point links to master where we continuously making changes